src/productcatalogservice/main.go

```diff
@@ -98,11 +99,34 @@ func initMeterProvider() *sdkmetric.MeterProvider {
 	mp := sdkmetric.NewMeterProvider(
 		sdkmetric.WithReader(sdkmetric.NewPeriodicReader(exporter)),
 		sdkmetric.WithResource(initResource()),
+		sdkmetric.WithView(			
+			sdkmetric.NewView(
+				sdkmetric.Instrument{Scope: instrumentation.Scope{Name: "go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"}},
+				sdkmetric.Stream{
+					AttributeFilter: allowedAttr(
```

Comment:
is it possible to do exclude list instead include ?

Reply:
Yes, I changed it to be an exclude list instead of an allow list 👍